src/main/java/uk/gov/hmcts/reform/sscs/config/RetryTemplateConfig.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.sscs.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.retry.backoff.FixedBackOffPolicy;
+import org.springframework.retry.support.RetryTemplate;
+
+@Configuration
+public class RetryTemplateConfig {
```

Comment:
Could this be used in the update and search classes rather than just the create class?